spec/lib/gxapi/google_analytics_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 module Gxapi
 
-  describe GoogleAnalytics do
+  xdescribe GoogleAnalytics do
```

Comment:
Yes, I was not mocking out the calls before because I wanted to develop it against the live API.  Mocking them at this point makes sense though.  Here is where the config lives for the integration tests (the path to the key is in the config file)

https://github.com/dlangevin/gxapi_rails/blob/master/spec/dummy/config/gxapi.yml